students/jstrauss123/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,19 @@
+# fizzbuzz - loop through 1 to 100, if # dvisible by 3 print fizz, if divisible by 5 print buzz if both, print fizzbuzz
+
+# iterate from 1 - 100
+for x in range(1, 101):
+	# test if value is divisible by 3 and 5 with no remainder
```

Comment:
5 spaces for indentions in Python - make sure your editor is set up to replace tabs with 4 spaces.